benchmarks/ccl_test_mconv.c

```diff
@@ -0,0 +1,46 @@
+#include "ccl.h"
```

Comment:
This test should be in python. The C tests are not reliably run and are basically deprecated.

Reply:
I've added a python test that does the same thing (`pyccl/tests/test_massdef.py`).

Reply:
OK. Let's remove the C test.